setup.cfg

```diff
@@ -4,7 +4,5 @@ dev = develop easy_install lasagne[testing]
 [pytest]
 addopts =
     -v --doctest-modules
-    --cov=lasagne --cov-report=term-missing
```

Comment:
Fair enough, I'll just get rid of that commit and use `skip-worktree` on my end so I can still debug.